src/docs/Components/Buttons/index.mdx

```diff
@@ -11,80 +11,80 @@ status:
+| Color scheme | Usage                                                                                   |
+| ------------ | --------------------------------------------------------------------------------------- |
+| Primary      | For main actions on the page or engaging actions                                        |
+| Primary-02   | For main specific actions dedicated to Campus                                           |
+| Neutral      | For secondary or minor actions on the page                                              |
+| Danger       | For buttons who the following actions is irrevocable, like deleting a product in a cart |
 
-### Three sizes to create hierarchy
+### Sizes
 
-There is **3 sizes** for the buttons to create hierarchy and for a better adaptability :
+There is 3 sizes for the buttons to create hierarchy and for a better adaptability.
 
-- Small
-- Medium
-- Large
+| Size                   | Value        |
+| ---------------------- | ------------ |
+| **Small**              | 2mu (32px)   |
+| **Medium** (_default_) | 3mu (48px)   |
+| **Large**              | 3.5mu (56px) |
 
 <Preview path="button-sizes" nude />
-
 <br />
 
-### A button width can either fit its content or its container
+### Styles
 
-<Preview path="button-widths" nude />
-
-<br />
+There is two styles to make what's important stand out:
 
-### Two styles to make what's important stand out
-
-- solid
-- bordered
+- Solid
+- Bordered
 
 <Preview path="intro" nude />
 
 ### Displaying buttons with icons
 
-In the button component, you can also add icons :
+In the button component, you can also add icons:
 
 - On the left
 - On the right
 - Icon only
 
 <Preview path="button-icon" nude />
 
-Icons are availables in different sizes depending on the button size. Here you can find which icon you can use for each button size :
+Icons are availables in different sizes depending on the button size. Here you can find which icon you can use for each button size:
+
+| **Button size** | **Icon only**                      | **Icon + text** |
+| --------------- | ---------------------------------- | --------------- |
+| **Small**       | 1.5mu (24px)                       | 1.5mu (24px)    |
+| **Medium**      | 2mu (32px)                         | 1.5mu (24px)    |
+| **Large**       | 2.5mu (48px) [_displayed in 40px_] | 2mu (32px)      |
 
-| **button size**        | **Icon S** | **Icon M**   | **Icon L** | **Icon XL**                        |
-| ---------------------- | ---------- | ------------ | ---------- | ---------------------------------- |
-| **small**              | 16px (1mu) | 24px (1.5mu) | /          | /                                  |
-| **medium** (_default_) | /          | 24px (1.5mu) | 32px (2mu) | /                                  |
-| **large**              | /          | /            | 32px (2mu) | 48px _(displayed in 40px)_ (2.5mu) |
+## Responsive behaviors
```

Comment:
it's not related to responsive, it's a variation, that can be applied to various screen sizes.

Reply:
fixed